### ISSUE TITLE
Other users' Minecraft data may get erased when switching versions

### ISSUE THREAD

**Issue by dktapps** [MCMrARM/mc-w10-version-launcher#99]
When two accounts on the same machine both have Minecraft installed, it has to be uninstalled from both accounts to allow the launcher to install its versions. While the launcher already backs up the Minecraft data path for the current user, it probably won't do the same thing for other user accounts, which could lead to loss of data.

**Comment by DamianXAkuma** [comment]
Yeah i installed MCLAuncher and now my Minecraft Windows 10 is gone and i have to repay for it in the shop any help to fix this?
